### PR TITLE
Use https:// instead of http://

### PR DESCRIPTION
**Pull request by nathanweeks**
Suggest standardizing on `https` instead of `http` for consistency.

### Check off the following:

- [X] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [X] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

File: content/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/authenticating-to-the-github-api.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ You can authenticate {% data variables.product.prodname_actions_runner_controlle
 
 1. Create a {% data variables.product.prodname_github_app %} that is owned by an organization. For more information, see "[AUTOTITLE](/apps/creating-github-apps/creating-github-apps/creating-a-github-app)". Configure the {% data variables.product.prodname_github_app %} as follows.
 
-   1. For "Homepage URL," enter `http://github.com/actions/actions-runner-controller`.
+   1. For "Homepage URL," enter `https://github.com/actions/actions-runner-controller`.
```
Comment (longca9396) [inline content/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/authenticating-to-the-github-api.md:31]: #https://github.com/longca9396/excelCPU

**Review by longca9396 — COMMENTED** [review]
nathanweeks:patch-1
[ghas_active_committers_Vi-t-Nam-Advertising_2024-02-03T1426.csv](https://github.com/github/docs/files/14154506/ghas_active_committers_Vi-t-Nam-Advertising_2024-02-03T1426.csv)

**Comment by nguyenalex836** [comment]
@nathanweeks Thanks so much for opening a PR! I'll get this triaged for review ✨

**Review by SiaraMist — APPROVED** [review]
Thanks for updating this, @nathanweeks! I'll go ahead and get this merged. 🚀

**Pull request merged**